source/connect/tableau.txt

```diff
@@ -31,7 +31,7 @@ listening, and click :guilabel:`Sign In` to connect.
 
 By default, :program:`mongosqld` binds to port ``3307``.
 
-.. figure:: /images/bi-connector/tableau.png
```

Comment:
Also, just caught that I messed up the *Server* field. It should probably say `localhost` like the others.